v2/cmd/wails/internal/commands/initialise/templates/templates/svelte/wails.tmpl.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "{{.ProjectName}}",
   "outputfilename": "{{.BinaryName}}",
+  "assetdir": "frontend/dist",
```

Comment:
Wails tries to infer the `assetdir` from the embed.FS out of main.go, which works fine as long as a user doesn't move the embed.FS out of main.go. We have seen several reports during the last few weeks where people had a working `wails dev` and `wails build` didn't, which was caused by having only adapted the `assetdir` and not changing the embed.FS.
Therefore I would suggest to remove the `assetdir` entry here, to avoid those inconsistencies.

Reply:
It means that the value of this field will be automatically obtained from the Go code without the need to fill in manually, right?

Reply:
Yeah, exactly.